netsniff-ng.c

```diff
@@ -810,8 +812,15 @@ static int next_multi_pcap_file(struct ctx *ctx, int fd)
 	} else
 		ftime = time(NULL);
 
-	slprintf(fname, sizeof(fname), "%s/%s%lu.pcap", ctx->device_out,
-		 ctx->prefix ? : "dump-", ftime);
+	if (ctx->overwrite_interval > 0) {
+	    slprintf(fname, sizeof(fname), "%s/%s%010lu.pcap", ctx->device_out,
+	             ctx->prefix ? : "dump-", ctx->file_number++);
+	    ctx->file_number = (ctx->file_number < ctx->overwrite_interval) ? ctx->file_number : 0;
+	}
+	else {
+        slprintf(fname, sizeof(fname), "%s/%s%lu.pcap", ctx->device_out,
+                 ctx->prefix ? : "dump-", ftime);
+    }
```

Comment:
Please consistently use tabs to indent. Currently it is a mixture of tabs and spaces.